packages/flutter/lib/src/painting/colors.dart

```diff
@@ -422,6 +422,9 @@ class ColorSwatch<T> extends Color {
   /// Returns an element of the swatch table.
   Color? operator [](T index) => _swatch[index];
 
+  /// Returns the valid keys for accessing operator[].
```

Comment:
nit: maybe rename this to `indices`? Since that's the terminology used by existing documentation (https://main-api.flutter.dev/flutter/painting/ColorSwatch-class.html), e.g.:

> The table is indexed by values of type T.

Reply:
I'd rather change the documentation since `index` suggests the keys are integers.  That already tripped me up.  What do you think about "The table is accessed with key values of type T."?

Reply:
Ah ok yeah calling the getter `indices` could be a bit misleading (although the word by itself doesn't seem to imply the key being an integer: https://en.wiktionary.org/wiki/index#Noun).

> An integer or other key indicating the location of data

Reply:
Fair point, seems that doesn't imply a number to everyone.  However if that was the preferred nomenclature that's what we'd use in Map which has the same semantics in its `operator[]`.

Reply:
I pushed a commit that updates the other documentation, PTAL.

Reply:
Those updates LGTM except there's one more reference to "index" in the doc on the constructor.